PCSX2/pcsx2
# [BUG] Fix a crash that happens when copying from one card to a folder memory card via BIOS

Fixes a crash when attempting to copy from one memory card to a folder memory card via the bios

(And maybe some other problems)

### Description of Changes
This fixes a crash that happens if you attempt to copy from one memory card to a formatted folder memory card

### Rationale behind Changes
I would like that to not happen

### Suggested Testing Steps
1. Have two memory cards (source being either a file or folder, the destination being a folder)
2. Make sure the destination (folder card) is formatted (doesn't seem to crash if it isn't)
3. Copy save data from the source to the destination folder via BIOS
4. Crash (or don't) :)

### P.S
This is probably a god awful fix and pull request and might've been better as a issue but I tried :)
